File: Java/12-Matrizes/Matriz.java
class Matriz {
    public static void main(String args[]) {
        //Matrizes são arrays de duas dimensões. Ideal para exibir tabelas.
        //para declarar uma matriz em java usamos

        int[][] matriz = new int[2][2]; //Temos uma matriz 2x2

        //Para prencher a matriz
        matriz[0][0] = 1;
        matriz[0][1] = 5;
        matriz[1][0] = 2;
        matriz[1][1] = 0;

        //Para exibir a matriz em forma de tabela precisamos de duas estruturas de repetição
        for(int i = 0; i < matriz.length; i++) { //Varre linhas
            for(int j = 0; j < matriz[i].length; j++) { //Varre colunas
                System.out.print(matriz[i][j] +" ");
            }
            System.out.println();
        }
    }
}

File: C/16-Matrizes/Matriz.c
#include<stdio.h>

int main(void)
{
    //Matrizes são arrays multidimensionais(normalmente 2 dimensões)
    //Para declarar uma matriz em C usamos:
    int num[2][2]; //matriz 2x2

    //Vamos definir os valores desta matriz
    num[0][0] = 4;
    num[1][0] = 2;
    num[0][1] = 1;
    num[1][1] = 3;

    //Para exibir uma matriz precisamos de uma estrutura de repetição aninhada
    //para linhas e colunas
    for(int linha = 0; linha < 2; linha++) {
        for(int coluna = 0; coluna < 2; coluna++) {
            printf("%d ",num[linha][coluna]);
        }
        //Para ficar em formato de tabela
        printf("\n");
    }

    return 0;
}
